src/nwb_benchmarks/benchmarks/time_remote_file_reading.py

```diff
@@ -144,6 +142,108 @@ def time_read_hdf5_pynwb_ros3(self, https_url: str):
         self.nwbfile, self.io, _ = read_hdf5_pynwb_ros3(https_url=https_url)
 
 
+class HDF5PyNWBFsspecHttpsPreloadedNoCacheFileReadBenchmark(BaseBenchmark):
+    """
+    Time the read of remote HDF5 NWB files using pynwb and fsspec with HTTPS with preloaded data without cache.
+    """
+
+    parameter_cases = parameter_cases
+
+    def setup(self, https_url: str):
```

Comment:
Will be interesting to see, but I suspect that the normal variability in network traffic may be larger than the difference in network traffic due to caching. But definitely worth taking a look.